### ISSUE TITLE
Is the customized layer supported?

### ISSUE THREAD

**Issue by wangerwei007** [NVIDIA-AI-IOT/torch2trt#52]
Some models have customized layer like corner_pool of cornernet . But how to write customized plugin to support it. The example plugin of interpolate has matched function in PyTorch(torch.nn.functional). It seems some difference as corner_pool layer. Could you give a example of customized plugin like corner_pool ?

**Comment by wangerwei007** [comment]
@jaybdub Thanks for  the response. I'm glad to provide it. For simplicity, I provide a easy model net for you(https://github.com/wangerwei007/PoolPlugins). I tried to write the plugin using torch2trt, and the Plugins to register successfully, but some errors have occurred, the code "eval(method)"  of "torch2trt.py" occurred that TopPool had Not defined. It seems some difference as the example
plugin of interpolate. So I look forward to giving me an example of customized layer like TopPool layer. Best wishes for you.

**Comment by SrivastavaKshitij** [comment]
Hi @jaybdub : I am facing a similar problem. 

A standard `torch.nn.BatchNorm2d` looks like : `<class 'torch.nn.modules.batchnorm.BatchNorm2d'> <bound method _BatchNorm.forward of BatchNorm2d(13, eps=1e-05, momentum=0.1, affine=True, track_running_stats=True)>`

I have a customized BN layer which uses the same exact parameters as a standard layer , however, its not a part of `torch.nn` module. 

`<class 'model.batchnorm.SynchronizedBatchNorm2d'> <bound method _SynchronizedBatchNorm.forward of SynchronizedBatchNorm2d(64, eps=1e-05, momentum=0.1, affine=True, track_running_stats=T
rue)>
`
How can I register this layer for conversion ?
